src/cutscenes/cutscene.c

```diff
@@ -478,3 +478,8 @@ void cutscene_enter(CallChain next, CutsceneID id) {
 	audio_bgm_play(res_bgm(cs->bgm), true, 0, 1);
 	eventloop_enter(st, cutscene_logic_frame, cutscene_render_frame, cutscene_end_loop, FPS);
 }
+
+const char *cutscene_get_name(CutsceneID id) {
+	const Cutscene *cs = g_cutscenes + id;
+	return cs->name;
```

Comment:
Not really important at all, but you could just write `return g_cutscenes[id].name;`. I'd also stick in an `assert((uint)id < NUM_CUTSCENE_IDS)` for good measure.